gems/faraday/2.5/faraday.rbs

```diff
@@ -112,6 +112,66 @@ module Faraday
     def patch: (?String | URI url, ?untyped body, ?untyped headers) ?{ (Faraday::Request) -> void } -> Faraday::Response
   end
 
+  class Error < StandardError
+    attr_reader response: Hash[Symbol, untyped]?
+    attr_reader wrapped_exception: Exception?
+
+    type exc = Exception | Hash[Symbol, untyped] | String
+
+    def initialize: (?exc? exc, ?Hash[Symbol, untyped]? response) -> void
+    def backtrace: () -> ::Array[String]?
+    def inspect: () -> ::String
+    def response_status: () -> untyped?
```

Comment:
Since `untyped` includes `nil`, simply `untyped` is fine.

Reply:
Thank you! 71ee7d3